templates/automotive/buyers-guide/index.html

```diff
@@ -0,0 +1,613 @@
+{% extends "automotive/base_automotive.html" %}
+
+{% block title %}Boost your automotive business with Ubuntu{% endblock %}
+{% block meta_description %}Ubuntu is widely used to reduce time to market in automotive companies, as a platform for data analytics, online services, robotics and machine learning. Infotainment, ADAS and cockpit application development on Ubuntu is faster. New real-time hypervisors enable Ubuntu VMs alongside functional safety RTOS solutions.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1NLnoNEP6cL46PotNGVuKLajehl-nfyBU4C8JT-FYMvQ/edit{% endblock meta_copydoc %}
+{% block body_class %}is-graphite{% endblock %}
+
+{% block content %}
+
+<section class="p-strip u-hide-on-reveal">
+  <div class="row">
+    <div class="col-6">
+      <h1><strong>Automotive OS <br class="u-hide--small" />Buyer's Guide</strong></h1>
+    </div>
+    <div class="col-6">
+      <h2>Choose the right Linux OS for in-vehicle systems</h2>
+      <p>Selecting the ideal Linux distribution for automotive applications involves various intricate facets. There are many crucial parameters that need consideration when making such choices. To accomplish this objective, our experts have compiled essential criteria that must be taken into account while picking a suitable Linux distribution for your automotive undertakings.</p>
+      <p><a href="#form" class="p-button--positive">Download the full guide</a></p>
+    </div>
+  </div>
+  <div class="u-fixed-width u-no-padding">
+    {{ image (
+      url="https://assets.ubuntu.com/v1/e9cf978f-rfq-_0001_GettyImages-1186091802.jpg",
+      alt="",
+      width="2496",
+      height="1066",
+      hi_def=True,
+      loading="auto",
+      attrs={"class": "banner-image--non-interactive"},
+      ) | safe
+    }}
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top is-shallow u-hide-on-reveal">
+  <hr class="is-fixed-width">
+  <div class="row">
```

Comment:
```suggestion
  <div class="u-fixed-width">
```
We should only use rows when there are contained columns

Reply:
AFAIK